client/src/lib/templates/LandingTemplate/LandingTemplate.svelte

```diff
@@ -0,0 +1,69 @@
+<script>
+  import books from "$lib/assets/books-circle.svg";
+  import flowChart from "$lib/assets/registration_flow_chart.svg";
+</script>
+
+<section class="banner fr-py-8w">
+  <div class="fr-container">
+    <div class="fr-grid-row">
+      <div class="fr-col-12 title-container">
+        <img class="books-image" alt="" src={books} />
+
+        <header class="fr-pl-3w">
+          <h1>Bienvenue sur le service de catalogage de données de l’État</h1>
+          <p>
+            Ce service permet de favoriser la circulation des données entre les
+            différentes administrations centrales et opérateurs de l’État.
+            L’intégration des différentes organisations se fait progressivement
+            et en fonction des demandes.
+          </p>
+        </header>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="fr-container fr-py-8w">
+  <div class="fr-grid-row">
+    <div class="fr-col-12">
+      <h3>Comment accéder aux catalogues ?</h3>
+
+      <p>
+        L’accès aux catalogues s’effectue via le compte datapass. Il est pour
+        l’instant possible uniquement aux agents des organisations enregistrées.
+        Pour en savoir plus sur les moyens de s’enregistrer et intégrer son
+        catalogue, vous pouvez consulter la documentation détaillée.
```

Comment:
Dans la maquette il y a ici un lien mais ici il n'y en a pas ; en effet une telle doc n'existe pas à ce jour. Est-ce qu'on indiquerait ceci en attendant... ? @DaFrenchFrog 


```suggestion
        catalogue, vous pouvez consulter la documentation détaillée (à venir).
```

Reply:
Pour moi ça se fera dans le cadre de #370

Reply:
Tu veux dire #359 ? Mon point était seulement que si on merge & déploie ceci, la page d'accueil à laquelle n'importe qui peut accéder sera temporairement un peu confuse car on se demande légitimement : "Ah bon, quelle doc, où est-elle ?"

Reply:
D'acc, je vais rajouter le (à venir) pour le moment